uk.co.compucorp.civicrm.tasksassignments/views/modal/document.html

```diff
@@ -8,6 +8,19 @@ <h4 class="modal-title">New Document</h4>
   </div>
   <div class="modal-body" nv-file-drop uploader="uploader">
     <div class="container">
+      <div class="row">
+        <div class="col-xs-12 col-sm-7"></div>
```

Comment:
I was using it to complete the 12 column bootstrap grid, will fix it by adding offset.